src/app/@ansyn/imagery-cesium/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.lib.json",
   "angularCompilerOptions": {
-    "enableIvy": true
+    "enableIvy": false
```

Comment:
why did you disable ivy?

Reply:
npm was not let you to publish package that compile with ivy,
so in production we change the ivy to false